File: NaruciBa/naruci_ba_mobile/lib/models/Poslovnica.dart
import 'package:json_annotation/json_annotation.dart';

part 'Poslovnica.g.dart';

@JsonSerializable()
class Poslovnica {
  final int poslovnicaID;
  final String adresa;
  final int? gradID;
  final String kontaktTel;
  final String kontaktEmail;
  final int? trgovackiLanacID;
  final DateTime? datumKreiranja;
  final DateTime? datumIzmjene;

  Poslovnica(
      {required this.poslovnicaID,
      required this.adresa,
      required this.gradID,
      required this.kontaktTel,
      required this.kontaktEmail,
      required this.trgovackiLanacID,
      required this.datumIzmjene,
      required this.datumKreiranja});

  /// Connect the generated [_$PersonFromJson] function to the `fromJson`
  /// factory.
  factory Poslovnica.fromJson(Map<String, dynamic> json) =>
      _$PoslovnicaFromJson(json);

  /// Connect the generated [_$PoslovnicaToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PoslovnicaToJson(this);
}

File: NaruciBa/NaruciBa.Model/Poslovnica.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NaruciBa.Model
{
    public class Poslovnica
    {
        public int PoslovnicaID { get; set; }
        public string Adresa { get; set; }
        public int? GradID { get; set; }
        public string KontaktTel { get; set; }
        public bool Status { get; set; }
        public string KontaktEmail { get; set; }
        public int? TrgovackiLanacID { get; set; }
        public DateTime? DatumKreiranja { get; set; }
        public DateTime? DatumIzmjene { get; set; }

        public virtual Grad Grad { get; set; }
        public virtual TrgovackiLanac TrgovackiLanac { get; set; }
    }
}
